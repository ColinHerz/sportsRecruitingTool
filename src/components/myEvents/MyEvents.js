import PropTypes from "prop-types";
import React from "react";

import "./myEvents.scss";

const MyEvents = props => {
	return (
		<main id="event">
			<h2>My Events</h2>

			<section id="controlled">
				<h3>Events I Control</h3>
				<ul>
					<li>Event 1</li>
					<li>Event 2</li>
					<li>Event 3</li>
				</ul>
			</section>

			<section id="in">
				<h3>Events I&apos;m In</h3>
				<ul>
					<li>Event 1</li>
					<li>Event 2</li>
					<li>Event 3</li>
				</ul>
			</section>
		</main>
	);
};

MyEvents.propTypes = {
	user: PropTypes.object.isRequired
};

export default MyEvents;
