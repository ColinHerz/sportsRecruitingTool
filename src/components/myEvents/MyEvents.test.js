/* eslint-env jest */
import { MemoryRouter } from "react-router-dom";
import React from "react";
import renderer from "react-test-renderer";

import MyEvents from "./MyEvents.js";

const genProps = () => {
	return {
		user: {}
	};
};

describe(`Rendering`, () => {
	it(`Renders beginning page correctly`, () => {
		const props = genProps();

		const component = renderer.create(
			<MemoryRouter>
				<MyEvents {...props} />
			</MemoryRouter>
		);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
