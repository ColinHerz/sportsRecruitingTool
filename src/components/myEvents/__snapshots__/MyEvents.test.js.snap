// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Renders beginning page correctly 1`] = `
<main
  id="my-events"
>
  <h2>
    My Events
  </h2>
  <button>
    <a
      href="/events/create/"
      onClick={[Function]}
    >
      Create Event
    </a>
  </button>
  <section
    id="in"
  >
    <h3>
      Events
    </h3>
    <ul />
  </section>
</main>
`;
